tests/std/tests/P0753R2_manipulators_for_cpp_synchronized_buffered_ostream/test.cpp

```diff
@@ -89,6 +92,76 @@ void test_osyncstream_manipulators(string_buffer<typename Ty::char_type, ThrowOn
     }
 }
 
+template <class Ty>
+class throwing_string_buffer : public basic_streambuf<Ty, char_traits<Ty>> {
+public:
+    throwing_string_buffer()  = default;
+    ~throwing_string_buffer() = default;
+
+    streamsize xsputn(const Ty*, streamsize) override {
+        throw std::ostream::failure("test exception");

```

Comment:
Style: I believe it would be clearer to use the true name of `ios_base::failure` (occurs below).

Also, we can construct this temporary with braces (I don't see any existing uses of parens for temporaries in this test, so we may as well begin using the modern style).